app/controllers/students_controller.rb

```diff
@@ -1,5 +1,32 @@
 class StudentsController < ApplicationController
+   
     def index
         @students = Student.paginate(page: params[:page], per_page: 10)
     end
-end
\ No newline at end of file
+
+    def show
+        @student = Student.find(params[:id])
```

Comment:
@Luke-kb  .. whenever you see yourself using a piece of code repetitively
it normally means you can dry it up by refactoring into another method.
You are loading the @student instance variable in 4 different methods.  show, edit, update, destroy

A common practice in controllers is to use a before_filter hook.
ref https://guides.rubyonrails.org/v3.2.13/action_controller_overview.html 
search for before_filter

```
# add this after the class declaration .. 
before_filter :load_student, except: [:index]

# then make a private method at the bottom of the class .. and remove all the other calls to
# @student = Student.find(params[:id])
 
def load_student
  @student = Student.find(params[:id])
end
```